drone-fly-integration-tests/pom.xml

```diff
@@ -76,5 +76,10 @@
       <artifactId>junit-jupiter-params</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>software.amazon.msk</groupId>
+      <artifactId>aws-msk-iam-auth</artifactId>
+      <version>1.1.1</version>
```

Comment:
same here, also add a property for it in the parent pom so we have the number in one place